test/jdk/java/net/Inet4Address/PingThis.java

```diff
@@ -32,6 +32,9 @@
  *          for InetAdress 0.0.0.0 and ::0
  * @run main PingThis
  * @run main/othervm -Djava.net.preferIPv4Stack=true PingThis
+ * @bug 8280498
+ * @summary InetAddress ::0 is not valid, InetAddress 0.0.0.0 only responds
+ *          when an external device reacts to 0.0.0.0
```

Comment:
Maybe that information could be added as a comment just before `return;` on line 52 instead:
`// On AIX, ...`

Reply:
I removed the @summary bit, as well as the reference to the bug, and moved the comments. However, I did not use // to start the comments - as I do not like 'mixed' comment formatting.